### PR TITLE
GRW-1281 Feature/add accordion block

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Link Accordion and AccordionItem to CMS

Screenshot 2022-08-04 at 17 41 20

## Justify why they are needed

This is base block for things like FAQ, menus, etc.  No design yet.  No markdown rendering for content yet

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] Will this be part of a product update? If yes, please write one phrase about this update.

[GRW-1281]: https://hedvig.atlassian.net/browse/GRW-1281?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by robinandeer — COMMENTED** [review]
@alebedev do you mind rebasing this PR on latest main -- will make it easier to review :)

File: apps/store/src/blocks/AccordionItemBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import * as Accordion from '@/components/Accordion/Accordion'
+import { SbBaseBlockProps } from '@/services/storyblok/storyblok'
+
+export type Props = SbBaseBlockProps<{
+  title: string
+  body: Richtext
+}>
+
+export const AccordionItemBlock = ({ blok }: Props) => {
+  const contentHtml = getStoryblokApi().richTextResolver.render(blok.body)
```
Comment (robinandeer) [inline apps/store/src/blocks/AccordionItemBlock.tsx:11]: is this an expensive calculation? consider memoizing it :)

**Review by robinandeer — APPROVED** [review]
Great stuff! That's a very simple integration with the rich text field type :)

File: apps/store/src/blocks/AccordionBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  )
+}
+
+const StyledRoot = styled(Accordion.Root)(({ theme }) => ({
+  padding: theme.space[4],
+}))
+
+const TitleWrapper = styled.div({
+  textAlign: 'center',
+})
```
Comment (guilhermespopolin) [inline apps/store/src/blocks/AccordionBlock.tsx:37]: nit

```suggestion
const StyledHeading = styled(Heading)({
  textAlign: 'center',
})
```

and then

```          
<Heading as="h2" variant="standard.20">
  {blok.title}
</Heading>
```

**Review by guilhermespopolin — COMMENTED** [review]
Well done :)

**Review by guilhermespopolin — APPROVED** [review]

**Pull request merged**